techniques/system/common/1.0/promises.st

```diff
@@ -190,7 +190,7 @@ bundle agent check_disable_agent
 
       # Only define this class when we're ready to die - this is a special class name in "abortclasses"
       "should_not_continue"
-        expression => "should_disable_agent",
+        expression => "should_disable_agent.!ignore_disable_agent",
```

Comment:
Hmm, just thinking about naming here. The command that uses this will be "rudder agent run --force", and we already have a "force_inventory" class to, well, force an inventory.

Can we name this force_agent_run instead? Or something similar, that starts with force_?